Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM golang:1.19 AS build
+FROM ghcr.io/acorn-io/images-mirror/golang:1.20-alpine AS build
 COPY / /src
 WORKDIR /src
 ARG TAG="v0.0.0-dev"
 ENV CGO_ENABLED=0
 RUN --mount=type=cache,target=/go/pkg --mount=type=cache,target=/root/.cache/go-build \
     go build -o bin/acorn-dns -ldflags "-s -w -X 'github.com/acorn-io/acorn-dns/pkg/version.Tag=${TAG}'" .
 
-FROM alpine:3.16.2 AS base
+FROM ghcr.io/acorn-io/images-mirror/alpine:latest AS base
```

Comment:
I think we'll want version tags published for this image too @yashgiri

Reply:
Hi @cjellick just wanted to confirm with you if it is fine to pick up alpine packages from this ghcr path.

Reply:
Meanwhile if needed this image can be used
`ghcr.io/acorn-io/images-mirror/alpine:3.16.2`